drivers/builtin/src/cipher.c

```diff
@@ -1578,7 +1574,7 @@ static int mbedtls_cipher_aead_decrypt(mbedtls_cipher_context_t *ctx,
 }
 #endif /* MBEDTLS_CIPHER_MODE_AEAD */
 
-#if defined(MBEDTLS_CIPHER_MODE_AEAD) || defined(MBEDTLS_NIST_KW_C)
+#if defined(MBEDTLS_CIPHER_MODE_AEAD)
 /*
  * Packet-oriented encryption for AEAD/NIST_KW: public function.
```

Comment:
```suggestion
 * Packet-oriented encryption for AEAD: public function.
```
here and also below for the `mbedtls_cipher_auth_decrypt_ext` function (sorry, but I cannot make the comment directly there)